scripts/template_upload_ec2.py

```diff
@@ -224,13 +246,17 @@ def run(**kwargs):
          **kwargs: Kwargs are passed by template_upload_all.
      """
     mgmt_sys = cfme_data['management_systems']
-    for provider in list_provider_keys('ec2'):
-        ssh_rhevm_creds = mgmt_sys[provider]['credentials']
-        username = credentials[ssh_rhevm_creds]['username']
-        password = credentials[ssh_rhevm_creds]['password']
-        upload_bucket_name = mgmt_sys[provider]['upload_bucket_name']
-        upload_template(provider, username, password, upload_bucket_name, kwargs.get(
-            'image_url'), kwargs.get('template_name'))
+    valid_providers = ['ec2east']
```

Comment:
@RonnyPfannschmidt I haven't gotten back around to this PR until now - can you elaborate on what you meant here? Looking back at my comment on the old commits, the intent here is to list provider keys here where we want to run and not run on all ec2 keys.